src/deploy/jenkins-uat.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('node-fetch');
+
+module.exports = () => {
+    const jenkinsUrl = "https://ctv-jenkins-02.truex.com/generic-webhook-trigger/invoke";
```

Comment:
Can we allow for a env variable overload here? 
```javascript
const jenkinsUrl = process.env.JENKINS_URL || "https://ctv-jenkins-02.truex.com/generic-webhook-trigger/invoke";
```

Reply:
this is a good idea

Reply:
The overloads don't seem too useful right now, if this is only going to be executed by the travis automation (and currently only does the defaults anyway.  But I can understand the sentiment of making it more future proof.

I wouldn't want to use environment variables for this though, and instead make it parameterized.

Reply:
For the Jenkins URL, it feels worth having a way to change it without requiring a code deploy (especially in this shared package given how many others it touches). Mainly based on the fact the Jenkins URL has changed in the past